cluster/environments/azure-aks/inputs.tf

```diff
@@ -26,3 +26,19 @@ variable "client_secret" {
 variable "ssh_public_key" {
   type = "string"
 }
+
+variable "flux_repo_url" {
+  type = "string"
```

Comment:
Should flux_repo_url have a default value?

Reply:
I will include the default value.